coordinator/src/services/translation/response/common.ts

```diff
@@ -228,34 +251,19 @@ export function translateAgentPerson(agentPerson: hl7V3.AgentPerson): Translated
         organization
       }
 
-      return translatedAgentPerson
-    } else {
-      const organization = createOrganization(representedOrganization)
-      const practitioner = createPractitioner(agentPerson)
-      const practitionerRole = createPractitionerRole(agentPerson, practitioner.id)
-      practitionerRole.organization = fhir.createReference(organization.id)
-
-      const healthCareProviderLicenseOrganization = representedOrganization.healthCareProviderLicense?.Organization
-      if (healthCareProviderLicenseOrganization) {
-        organization.partOf = {
-          identifier: getOrganizationCodeIdentifier(healthCareProviderLicenseOrganization.id._attributes.extension),
-          display: healthCareProviderLicenseOrganization.name?._text
-        }
-      }
-
-      const translatedAgentPerson: TranslatedAgentPerson = {
-        practitionerRole,
-        practitioner,
-        healthcareService: null,
-        locations: [],
-        organization
-      }
-
       return translatedAgentPerson
     }
   }
 }
 
+function shouldHavePrimaryCareFormat(prescriptionType?: string): boolean {
+  if (prescriptionType?.startsWith("01", 0)) {
```

Comment:
this can be simplified a lot
`return prescriptionType?.startsWith("01", 0)`

Reply:
My bad, lazy c&p